pkg/cli/admin/release/mirror.go

```diff
@@ -847,19 +869,24 @@ func (o *MirrorOptions) Run() error {
 	return nil
 }
 
-// printImageContentInstructions provides examples to the user for using the new repository mirror
+type mirrorSet struct {
+	source  string
+	mirrors []string
+}
+
+// printImageMirrorInstructions provides examples to the user for using the new repository mirror.
 // https://github.com/openshift/installer/blob/master/docs/dev/alternative_release_image_sources.md
-func printImageContentInstructions(out io.Writer, from string, toList []string, signatureToDir string, repositories map[string]struct{}) error {
-	type installConfigSubsection struct {
-		ImageContentSources []operatorv1alpha1.RepositoryDigestMirrors `json:"imageContentSources"`
-	}
+func printImageMirrorInstructions(out io.Writer, from string, toList []string, signatureToDir string, repositories map[string]struct{}, printImageSourceInstructions string) error {
 
-	var sources []operatorv1alpha1.RepositoryDigestMirrors
+	var (
```

Comment:
I think, there is no shortcut return if `printImageSourceInstructions` is none.

Reply:
fixed.